src/settings/MappingProfiles/MappingProfilesTransformationsModal/MappingProfilesTransformationsModal.js

```diff
@@ -1,52 +1,61 @@
 import React, {
   useState,
+  useCallback,
   useEffect,
 } from 'react';
 import PropTypes from 'prop-types';
 import { FormattedMessage } from 'react-intl';
-import { noop } from 'lodash';
+import {
+  get,
+  noop,
+} from 'lodash';
 
 import {
   Button,
-  Icon,
   Modal,
   Pane,
   Paneset,
-  SearchField,
-  AccordionSet,
-  Accordion,
-  FilterAccordionHeader,
-  Checkbox,
 } from '@folio/stripes/components';
 import {
   ExpandFilterPaneButton,
   CollapseFilterPaneButton,
 } from '@folio/stripes/smart-components';
 
+import { recordTypes } from '../RecordTypeField';
 import { TransformationsForm } from './TransformationsForm';
-import { mappingProfileTransformations } from './TransformationsField/transformations';
+import { SearchForm } from './SearchForm';
 
 import css from './MappingProfilesTransformationsModal.css';
 
+const initialSearchFormValues = { filters: { recordTypes: recordTypes.map(recordType => recordType.value) } };
+const fullWidthStyle = { style: { width: '100%' } };
```

Comment:
I thought about this, but It won't work, as `Pane` does not support custom class prop and in case of providing the override one the styles which is [used internally](https://github.com/folio-org/stripes-components/blob/master/lib/Pane/Pane.css#L3) would be lost and thus styles with it.